src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,14 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
 public class Country implements Cloneable {
```

Comment:
The `Cloneable` interface is implemented here, but there is no actual need to clone entities in typical Hibernate usage. Hibernate handles entity instances and their persistence, and cloning entities can lead to unexpected behavior, especially with managed entities and their lifecycle. Unless there is a specific requirement that necessitates the cloning of `Country` entities, it would be better to remove the `Cloneable` interface and the `clone` method to avoid potential issues.